matlab-plugin-agent/src/main/java/com/mathworks/ci/RunMatlabBuildService.java

```diff
@@ -35,7 +40,14 @@ public ProgramCommandLine makeProgramCommandLine() throws RunBuildException {
 
         ProgramCommandLine value;
         try {
-            value = this.runner.createCommand(getContext(), "buildtool " + getTaskName());
+            String cmd = "buildtool";
+            if (!getTaskName().isEmpty()){
```

Comment:
this could potentially lead to bug which will lead to swapping of task and build options